Backend/src/routers/analytics.py

```
@@ -116,38 +116,58 @@ async def get_class_performance(
 # School comparison
 @router.get("/school-comparison")
 async def get_school_comparison(
-    current_user: User = Depends(require_admin),
+    current_user: User = Depends(get_current_active_user),
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)